cluster/manifests/prometheus/ingress.yaml

```diff
@@ -1,3 +1,4 @@
+# TODO: Remove
```

Comment:
^

Reply:
I want to deploy RouteGroup and remove Ingress in the followup PR such there is no down time (couple of seconds possible between deletion and Skipper discovering route group).

Reply:
There is nothing that depend on it that can not deal with a downtime, but if you want to make it 0-downtime it's also fine.